src/ray/object_manager/plasma/eviction_policy.h

```diff
@@ -92,13 +91,18 @@ class IEvictionPolicy {
   ///        be stored into this vector.
   /// \return The total number of bytes of space chosen to be evicted.
   virtual int64_t ChooseObjectsToEvict(int64_t num_bytes_required,
-                                       std::vector<ObjectID> *objects_to_evict) = 0;
+                                       std::vector<ObjectID> &objects_to_evict) = 0;
 
   /// This method will be called when an object is going to be removed
   ///
   /// \param object_id The ID of the object that is now being used.
   virtual void RemoveObject(const ObjectID &object_id) = 0;
 
+  /// Returns whether the object exists or not.
+  ///
+  /// \param object_id The ID of the object.
+  virtual bool IsObjectExists(const ObjectID &object_id) const = 0;
```

Comment:
this api is a bit confusing. i.e. does this mean evictable?  since this only used in test, consider make it private and use FRIEND_TEST to access it.

Reply:
Done. moved to private.